specification/frontdoor/resource-manager/Microsoft.Network/stable/2021-06-01/examples/FrontdoorRulesEngineCreate.json

```diff
@@ -0,0 +1,281 @@
+                "redirectType": "Moved",
+                "redirectProtocol": "HttpsOnly",
+                "customHost": "www.bing.com",
+                "customPath": "/api",
+                "customFragment": "fragment",
+                "customQueryString": "a=b"
+              }
+            },
+            "matchConditions": [
+              {
+                "rulesEngineMatchVariable": "RemoteAddr",
+                "rulesEngineOperator": "GeoMatch",
+                "rulesEngineMatchValue": [
+                  "CH"
+                ]
+              }
+            ],
+            "matchProcessingBehavior": "Stop"
+          },
+          {
+            "name": "Rule2",
+            "priority": 2,
+            "action": {
+              "responseHeaderActions": [
+                {
+                  "headerActionType": "Overwrite",
+                  "headerName": "Cache-Control",
+                  "value": "public, max-age=31536000"
+                }
+              ]
+            },
+            "matchConditions": [
+              {
+                "rulesEngineMatchVariable": "RequestFilenameExtension",
+                "rulesEngineOperator": "Equal",
+                "rulesEngineMatchValue": [
+                  "jpg"
+                ],
+                "transforms": [
+                  "Lowercase"
+                ]
+              }
+            ]
+          },
+          {
+            "name": "Rule3",
+            "priority": 3,
+            "action": {
+              "routeConfigurationOverride": {
+                "@odata.type": "#Microsoft.Azure.FrontDoor.Models.FrontdoorForwardingConfiguration",
+                "customForwardingPath": null,
+                "forwardingProtocol": "HttpsOnly",
+                "cacheConfiguration": {
+                  "queryParameterStripDirective": "StripOnly",
+                  "queryParameters": "a=b,p=q",
+                  "dynamicCompression": "Disabled",
+                  "cacheDuration": "P1DT12H20M30S"
+                },
+                "backendPool": {
+                  "id": "/subscriptions/subid/resourceGroups/rg1/providers/Microsoft.Network/frontDoors/frontDoor1/backendPools/backendPool1"
+                }
+              }
+            },
+            "matchConditions": [
+              {
+                "rulesEngineMatchVariable": "RequestHeader",
+                "rulesEngineOperator": "Equal",
+                "rulesEngineMatchValue": [
+                  "allowoverride"
+                ],
+                "transforms": [
+                  "Lowercase"
+                ],
+                "negateCondition": false,
+                "selector": "Rules-Engine-Route-Forward"
+              }
+            ]
+          }
+        ]
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/subid/resourceGroups/rg1/providers/Microsoft.Network/frontDoors/frontDoor1/rulesEngines/rulesEngine1",
+        "name": "rulesEngine1",
```

Comment:
>name [](http://example.com/codeflow?start=9&length=4)

You're missing the type on these. Should be `Microsoft.Network/frontDoors/rulesEngines`

Reply:
Updated the type

Reply:
A bunch of the rulesEngines examples had the issue btw.

Reply:
Fixed the issue with all rulesEngine examples